src/Components/v2/ArtistCard.tsx

```diff
@@ -54,6 +55,17 @@ export class ArtistCard extends React.Component<Props> {
   }
 }
 
+// Why not use <Truncator>? Because with only one line for the artist origin/
+//  birth year, the Truncator prematurely truncates, showing "American, b. 1938"
+//  as "American, b. ...".
```

Comment:
This probably doesn't require a comment as we use `text-overflow` in quite a few places. The truncator is specific to large bodies of text + read more.

Reply:
What do you think about me adding this JSDoc comment to the `Truncator` component, instead?

```
/**
 * Truncate large bodies of text.
 * If you need to truncate a single line, consider using `text-overflow: ellipsis`.
 */
export const Truncator: React.SFC<Props> = ({
....
```

I would see this as useful guidance for someone, like me, who was uncertain about when to use the Truncator and when to do it with plain CSS.

Reply:
That's better 👍